go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/redhat-appstudio/jvm-build-service
 
-go 1.21
+go 1.22.0
```

Comment:
Won't this also require an updated Dockerfile with the new go version?

Reply:
Ah, yes, I'd missed there was a go reference in the top level Dockerfile.

Reply:
Although, whether go1.22 is available is another question...